paramak/shape.py

```diff
@@ -739,7 +756,7 @@ def export_stl(self, filename, tolerance=0.001):
 
         return str(path_filename)
 
-    def export_stp(self, filename=None, units='mm'):
+    def export_stp(self, filename=None, units='mm', solid_or_wire='solid'):
```

Comment:
```suggestion
    def export_stp(self, filename=None, units='mm', mode='solid'):
```

Same as above, could solid_or_wire be replaced by mode ?